availability/watcher.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import datetime as dt
+import json
+import logging
+import sys
+import threading
+import time
+
+import queue
+import requests
+
+from availability import config
+
+
+SERVICES = config.get_config().get("services")
+ELASTIC = config.get_config().get("elastic", "http://127.0.0.1:9200")
+SERVICE_CONN_TIMEOUT = config.get_config().get("connection_timeout", 1)
+SERVICE_READ_TIMEOUT = config.get_config().get("read_timeout", 10)
+
+LOG_LEVEL = config.get_config().get("logging", {}).get("level", "INFO")
+LOG = logging.getLogger("watcher")
+LOG.setLevel(LOG_LEVEL)
+
+
+def check_availability(data, results_queue):
+    """Check if service is available and put result to Queue.
+
+    :param data: dict that represents service query data
+    :param results_queue: Queue for results
+    """
+    try:
+        requests.get(data["url"], timeout=(SERVICE_CONN_TIMEOUT,
+                                           SERVICE_READ_TIMEOUT))
+        data["status"] = True
+    except requests.exceptions.RequestException:
```

Comment:
It would be nice to log warning in case if something goes down with information about exception